docs/modules/commons/pages/expressions.adoc

```diff
@@ -454,8 +454,14 @@ https://regex101.com/[Regex101] can be used to test and debug regular expression
 ----
 
 * *$regularExpression* - the regular expression to match substring(s)
-* *$replacement* - the replacement string, it may contain references to captured
-substrings, e.g. `$1` is a reference to the first group
+* *$replacement* - the replacement string
++
+[NOTE]
+====
+Backslashes (`\`) and dollar signs (`$`) in the replacement string may cause the results to be different than if it were
```

Comment:
I think it makes sense to add examples

Reply:
we have examples in the table below, probably, people do not read it